src/tests/contract_test/wallet_test.cljs

```diff
@@ -65,3 +65,15 @@
     (fn []
       (p/let [response (contract-utils/call-rpc "wallet_getEthereumChains")]
         (assert-ethereum-chains response)))))
+
+(defn assert-search-ens
+  [response]
+  (is (= "0xeefb13c7d42efcc655e528da6d6f7bbcf9a2251d" response)))
+
+(deftest wallet-search-ens-test
+  (h/test-async :wallet/search-ens
+    (fn []
+      (p/let [input     "test.eth"
+              chain-id  constants/ethereum-mainnet-chain-id
+              ens-address (contract-utils/call-rpc "ens_addressOf" chain-id input)]
+        (assert-search-ens ens-address)))))
```

Comment:
Hey @FFFra , thanks for this PR

I think there are more cases to tests, for example giving a `*.stateofus.eth`  address. atm I only know about the existance of `qoqobolo.stateofus.eth` on goerli.

We can also test the resolution on Sepolia and also the response for an invalid ENS.

wdyt?